playground/deepspeed_config_s2.json

```diff
@@ -1,12 +1,34 @@
 {
-  "zero_optimization": {
-    "stage": 2,
-    "offload_optimizer": {
-      "device": "cpu"
+  "bf16": {
+    "enabled": "auto"
+  },
+  "optimizer": {
+    "type": "AdamW",
+    "params": {
+      "lr": "auto",
+      "betas": [
+        0.9,
+        0.999
+      ],
+      "eps": "auto",
+      "weight_decay": "auto"
```

Comment:
Unfortunately, I believe defining an optimizer doesn't work after Transformers version 4.30.0, would you be able to remove this?

Reply:
I have reset the config to what is on your repo.